### PR TITLE
Remove ML-powered queries

### PR DESCRIPTION
**Pull request by henrymercer**
Removes the code to load ML-powered queries since this feature is now fully disabled.  For more information, including details on GitHub's investment in AI-powered security technology, see ["CodeQL code scanning deprecates ML-powered alerts."](https://github.blog/changelog/2023-09-29-codeql-code-scanning-deprecates-ml-powered-alerts/)

### Merge / deployment checklist

- [x] Confirm this change is backwards compatible with existing workflows.
- [x] Confirm the [readme](https://github.com/github/codeql-action/blob/main/README.md) has been updated if necessary.
- [x] Confirm the [changelog](https://github.com/github/codeql-action/blob/main/CHANGELOG.md) has been updated if necessary.

### CODE REVIEW

File: src/config-utils.ts [hunk 1]
```diff
@@ -1021,35 +978,29 @@ async function addQueriesAndPacksFromWorkflow(
...
   // "+" means "don't override config file" - see shouldAddConfigFileQueries
   queriesInput = queriesInput.replace(/^\+/, "");
 
   for (const query of queriesInput.split(",")) {
-    const didInject = await parseQueryUses(
+    await parseQueryUses(
       languages,
       codeQL,
       resultMap,
-      packs,
```
Comment (adityasharad) [inline src/config-utils.ts:1041]: Why don't we pass along packs here? What happens if the workflow specifies `packs` too?

**Review by adityasharad — COMMENTED** [review]
Nice! One question only.

File: src/config-utils.ts (hunk 1 above)
Comment (henrymercer) [reply src/config-utils.ts:1041]: Previously this was only used to inject the ML-powered pack when appropriate.  Augmenting the config with the `packs` input happens in `calculateAugmentation`.  I'll update the name to `addQueriesFromWorkflow` to make this clearer.

**Review by adityasharad — APPROVED** [review]

**Comment by adityasharad** [comment]
Removed the ML-powered queries checks from branch protection.

**Pull request merged**